identity-service/src/routes/cognito.js

```diff
@@ -85,6 +85,10 @@ module.exports = function (app) {
         // address is always present but nullable
         // name is always present but nullable
         const { id_number: idNumber, phone, date_of_birth: dob, address, name } = userInfo
+        const nameLowercased = name
+          // if name is not null, then first and last are always present according to api
+          ? { first: name.first.toLowerCase(), last: name.last.toLowerCase() }
```

Comment:
I see the comment, but can we be extra extra safe by using `name.first?.toLowerCase()` in case the API lies 👀

Reply:
yes